.github/CODEOWNERS

```diff
@@ -10,18 +10,19 @@
 # This file
 /.github/CODEOWNERS @edolstra
 
-# Boostraping and core infra
```

Comment:
I made this category, but then "Nixpkgs Internals" was made, and I don't find this one sufficiently different from that one to warrant keeping it around.